src/pixi-multistyle-text.ts

```diff
@@ -40,103 +54,137 @@ interface TextStyle {
 }
 
 export interface TextStyleExtended extends TextStyle {
-	valign?: "top" | "middle" | "bottom" | "baseline" | number;
-	debug?: boolean;
-	tagStyle?: "xml" | "bbcode";
+  valign?: VAlign;
+  debug?: boolean;
+  tagStyle?: TagStyle;
 }
 
-export interface TextStyleSet {
-	[key: string]: TextStyleExtended;
+export interface TextStyleExtendedWithDefault extends TextStyleExtended {
```

Comment:
Can we use `extends Required<TextStyleExtended>` while overiding some of them to keep them optional?

Otherwise, does it really make sense to have a type like this instead of using `TextStyleExtended`?  I've read the rest of the code and it doesn't seem to really provide any value?